cypress/integration/unit_tests/checkbox_test.js

```diff
@@ -463,6 +465,69 @@ describe('Unit test for toggleLayerOn', () => {
     // All images now shown on page.
     expectBlobImageCount(2 * tilesOnMap);
   });
+
+  it('tests json url', () => {
+    let map = null;
+    createGoogleMap()
+        .then((returnedMap) => map = returnedMap)
+        .then(
+            () => map.fitBounds(new google.maps.LatLngBounds(
+                new google.maps.LatLng({lat: 26, lng: -97.4}),
+                new google.maps.LatLng({lat: 27.62, lng: -97.1}))));
+
+    expectNoBlobImages().then(
+        () => addLayer(
+            {
+              'ee-name': 'tile_asset',
+              'asset-type': LayerType.MAP_TILES,
+              'urls': [
+                'https://storms.ngs.noaa.gov/storms/tilesd/services/tileserver.php?/20170827-rgb.json',
+              ],
+              'display-name': 'tiles',
+              'display-on-load': true,
+              'index': 4,
+            },
+            map));
+    // Experimentally found.
+    expectBlobImageCount(2);
+  });
+
+  it('tests slow json url with toggle off and on', () => {
+    let map = null;
+    createGoogleMap()
+        .then((returnedMap) => map = returnedMap)
+        .then(
+            () => map.fitBounds(new google.maps.LatLngBounds(
+                new google.maps.LatLng({lat: 26, lng: -97.4}),
+                new google.maps.LatLng({lat: 27.62, lng: -97.1}))));
+    let resolveFunction = null;
```

Comment:
same as just let resolveFunction ?

Reply:
intellij on my mac complains for me sometimes if not initialized, because it doesn't know it gets set immediately in the Promise lambda. But seems ok now, so removed.